tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/reindexing/NodesSecondaryChildrenRelatedTests.java

```diff
@@ -211,6 +219,8 @@ protected FileModel createDocument(String filename)
                 .createContent(new FileModel(filename, FileType.TEXT_PLAIN, "content"));
         }
 
+
+
```

Comment:
Random extra new lines added :rocket: 

We should look into automatic formatting :slightly_smiling_face: